airflow/models.py

```diff
@@ -1961,6 +1965,9 @@ class derived from this one results in the creation of a task object,
     :type resources: dict
     :param run_as_user: unix username to impersonate while running the task
     :type run_as_user: str
+    :type notifiers: dict
+    :param notifiers: A dict of AirflowNotifiers which are called whenever the
```

Comment:
I've made it a dict to be able to specify a particular notifier in the `send_notification` method. It has a `notifier` argument. Which is the key of the notifiers dict.
I use it to provide backwards compatibility in the `email_alert` method, which calls `send_notification` which `notifier="email"`